lib/src/visitor/serialize.dart

```diff
@@ -802,8 +822,9 @@ class _SerializeCssVisitor
   /// Writes indentation based on [_indentation].
   void _writeIndentation() {
     for (var i = 0; i < _indentation; i++) {
-      _buffer.writeCharCode($space);
-      _buffer.writeCharCode($space);
+      for (var j = 0; j < _indentWidth; j++) {
+        _buffer.writeCharCode(_indentCharacter);
```

Comment:
It looks like this will use two tabs for indentation if `indentType: 'tab'` is passed with the default width. Is that what node-sass does?